Update configuration.md to include psql setup

Stoner19 opened the pull request:
PostgreSQL initial setup added

Stoner19 committed:
Update configuration.md

PostgreSQL initial setup added

saleh-old commented:
Thank you for contributing. I appreciate it.

Right before I saw your PR I pushed these instructions for [Ubuntu](https://docs.jesse-ai.com/docs/getting-started/environment-setup.html#ubuntu). But I didn't add a proper description. Yours is great. Could you please move it there? 

The thing is I'm not sure if these commands work for macOS and Windows too. Also, please mention that in the docker image, a default database and user are created too so they don't need to do it there".

Stoner19 committed:
PostgreSQL setup updated

PostgreSQL setup instructions for Ubuntu moved to the proper page, as requested.

Stoner19 commented:
PostgreSQL setup moved to the `Ubuntu` heading with a note about docker image set up.

As far as OSX and Windows goes, I don't know about those. I can probably test the initial psql setup on my Mac, but windows I definitely won't be of any help.

saleh-old commented:
Thanks. It's OK, macOS would be good too.
